CHANGELOG.md

```diff
@@ -1,2 +1,3 @@
 - Fix bug where functions:config get command failed on empty variable. (#4707)
 - Fix invoker type mismatch for https functions.
+- Add features for Auth Alignment in Emulator UI
```

Comment:
Some more specificity would be nice here - i think `auth alignment` isn't something that an external user would understand, and these notes get tweeted out.

Reply:
Updated in https://github.com/firebase/firebase-tools/pull/4747